### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by Boomchil**
Hi !

I used to love GHH, and once it got doomed I discovered Gloomhaven Secretary, which is better in every way ! Here is a full french translation for anyone interested, plus some typo I stumbled upon while translating it.

Note : For the "self" word, I had to duplicate it for "female" monsters, as in french we got different "self" words for male and female.

Hope you can merge it in the main repository so my french fellow can use this wonderful app ! 

Thank you

### CODE REVIEW

**Comment by Lurkars** [comment]
First of all thank you so much for this work! I will take a deeper look later. I also have some small updates in the pipe which add some label, so would be awesome if you can migrate too then. 

Some important information about Git(Hub): if you just commit new changes to your branch of this PullRequest, the PR should update automatically!

Now for the self target, first: is the a physical french translation with differences on the card? I must say, that I don't like the idea of adding two different values for the same meaning, so maybe you can just set the label to `Sur lui-/elle-même` or `Sur lui-même/elle-même`?

Thanks again for your work!

File: data/jotl/deck/black-sludge.json [hunk 1]
```diff
@@ -129,7 +129,7 @@
           "subActions": [
             {
               "type": "specialTarget",
-              "value": "self"
+              "value": "selfFemale"
```
Comment (Lurkars) [inline data/jotl/deck/black-sludge.json:132]: Would like to keep it on "self" and us single label, besides, `selfFemale` is referenced here, in label files it's `self2`.

File: src/assets/locales/fr.json [hunk 2]
```diff
@@ -0,0 +1,710 @@
...
+      "summon": "Invoque",
+      "swing": "Swing",
+      "target": "Cible",
+      "targets": "Cibles",
+      "teleport": "Téléporte",
+      "trigger": "Déclenchement"
+    },
+    "specialTarget": {
+      "self": "Sur lui-même",
+      "self2": "Sur elle-même",
```
Comment (Lurkars) [inline src/assets/locales/fr.json:549]: `self2` should be removed for a single value like `Sur lui-/elle-même`. Besides `selfFemale` is used in data and both label are missing in other languages.

**Review by Lurkars — CHANGES_REQUESTED** [review]
Thanks for this. The typos and wrong label in level are already changed in main branch from my side. Please merge latest main into your branch and update the `self` label stuff. Then I can merge this! Thank you again.

**Comment by Boomchil** [comment]
Hi, just dropped the SelfFemale as you want, even though it's used in the official French translation, it's used just for the Black Sludge, it'll be alright :)

**Comment by Lurkars** [comment]
Thank you very much, will merge later today!

**Comment by Lurkars** [comment]
@Boomchil Hey thanks again for your work. Do you mind if I ping you in commits, where I added some label? For now, I tried to do the french part on my own for the recent changes, but first of all a recheck will be good and I may also run into issues where I cannot adopt the text from your prev. translations.

**Comment by Boomchil** [comment]
Hi 👋 

Please do, it will make my life easier as I planned to do a check every now and then to see if new labels had been added ☺️.

**Comment by Lurkars** [comment]
> Hi 👋 
> 
> Please do, it will make my life easier as I planned to do a check every now and then to see if new labels had been added ☺️. 

@Boomchil did the mention in latest commit worked for you or do I need to mention you explicit here in comment?

**Comment by Boomchil** [comment]
I git an email for the mention, it works fine ;)

**Pull request merged**